modules/test/playwright/helpers/ApiHelpers.ts

```diff
@@ -64,7 +64,7 @@ export class ApiHelpers {
 			headers: await this.getHeader(),
 		});
 
-		return response.json();
+		return response;
```

Comment:
We are changing this for all. I don't think it's correct. Can we do a check? Like
* if it's a json return the response.json() as before
* otherwise return only the response

Reply:
New implementation:

```
		const response = await this.page.request.patch(url, {
			data,
			headers: await this.getHeader(),
		});

		const text = await response.text();

		if (!text) {
			return response;
		}

		return response.json();
```